modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -420,6 +420,20 @@
 	}
 }
 
```

Comment:
Hey @gabrielpramos , we need to specify more this selector. I mean, we need to use some Calendar specific identifier to avoid changing/interfering with the style from other portlets.

Thx